src/python/pants/backend/python/targets/unpacked_whls.py

```diff
@@ -47,25 +48,31 @@ def __init__(
-                               you want to unpack
-        :param list include_patterns: fileset patterns to include from the archive
-        :param list exclude_patterns: fileset patterns to exclude from the archive. Exclude patterns
-          are processed before include_patterns.
-        :param compatibility: Python interpreter constraints used to create the pex for the requirement
-                              target. If unset, the default interpreter constraints are used. This
-                              argument is unnecessary unless the native code depends on libpython.
-        :param str within_data_subdir: If provided, descend into '<name>-<version>.data/<subdir>' when
-                                       matching `include_patterns`. For python wheels which declare any
-                                       non-code data, this is usually needed to extract that without
-                                       manually specifying the relative path, including the package
-                                       version. For example, when `data_files` is used in a setup.py,
-                                       `within_data_subdir='data'` will allow specifying
-                                       `include_patterns` matching exactly what is specified in the
-                                       setup.py.
-        """
+    :param str module_name: The name of the specific python module containing headers and/or
+                            libraries to extract (e.g. 'tensorflow').
+    :param list libraries: addresses of python_requirement_library targets that specify the wheels
+                           you want to unpack
+    :param list include_patterns: fileset patterns to include from the archive
+    :param list exclude_patterns: fileset patterns to exclude from the archive. Exclude patterns
+      are processed before include_patterns.
+    :param compatibility: Python interpreter constraints used to create the pex for the requirement
+                          target. If unset, the default interpreter constraints are used. This
+                          argument is unnecessary unless the native code depends on libpython.
+    :param bool within_data_subdir: If True, descend into '<name>-<version>.data/' when matching
+                                    `include_patterns`. For python wheels which declare any non-code
+                                    data, this is usually needed to extract that without manually
+                                    specifying the relative path, including the package version. For
+                                    example, when `data_files` is used in a setup.py,
+                                    `within_data_subdir=True` will allow specifying
+                                    `include_patterns` matching exactly what is specified in the
+                                    setup.py.
+    """
```

Comment:
These should be indented. Likely, you'll be able to select all the lines and press tab once to fix it quickly (that worked in most cases for me).

Reply:
Fixed.